### ISSUE TITLE
Pluggable policy engine for attestation server

### ISSUE THREAD

**Issue by bpradipt** [confidential-containers/attestation-service#19]
Currently the attestation server is tightly couple with Open Policy Agent (OPA).
It would be useful to make it modular and enable usage of alternative policy engines like [kyverno](https://kyverno.io).

**Comment by jialez0** [comment]
> Currently the attestation server is tightly couple with Open Policy Agent (OPA). It would be useful to make it modular and enable usage of alternative policy engines like [kyverno](https://kyverno.io).

@bpradipt It will be easy to realize the modularization of the policy engine (only need to do some additional encapsulation for the policy verification part), but we may need to redesign the message format of AS interacting with the policy engine to make it universal enough to support various policy engines. In the subsequent PR, perhaps we can achieve this.

**Comment by jialez0** [comment]
@bpradipt Hi, after #24 be merged, I will raise a new PR to pluggable policy engine :)

**Comment by bpradipt** [comment]
> @bpradipt Hi, after #24 be merged, I will raise a new PR to pluggable policy engine :)

awesome @jialez0 :-)

**Comment by jialez0** [comment]
Hi, @bpradipt , I have raise the PR #25  to solve this issue. Now you can review it.
